src/pages/faq-page.html

```diff
@@ -0,0 +1,159 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/marked-element/marked-element.html">
+<link rel="import" href="../elements/shared-styles.html">
+<link rel="import" href="../mixins/scroll-functions.html">
+
+<dom-module id="faq-page">
+    <template>
+        <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+        <style>
+             :host {
+                display: block;
+            }
+
+            .content-wrapper {
+                background-color: #4527A0;
+                background-image: linear-gradient(to right, #7E57C2, #4527A0);
```

Comment:
should be generic as you config all colors in `shared-styles.html`. 
Have you tried a gray color instead?